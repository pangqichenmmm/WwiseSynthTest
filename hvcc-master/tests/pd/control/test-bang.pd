#N canvas -1628 83 350 293 10;
#X obj 34 34 loadbang;
#X obj 34 236 print;
#X obj 34 196 b;
#X obj 74 196 bng 15 250 50 0 empty empty empty 17 7 0 10 -262144 -1
-1;
#X obj 104 196 bang;
#X msg 144 196 bang;
#X msg 34 114 22 3 1;
#X obj 100 114 f 1.45;
#X msg 158 114 hello;
#X obj 34 74 t b b b;
#X obj 34 156 t a a a a;
#X connect 0 0 9 0;
#X connect 2 0 1 0;
#X connect 3 0 1 0;
#X connect 4 0 1 0;
#X connect 5 0 1 0;
#X connect 6 0 10 0;
#X connect 7 0 10 0;
#X connect 8 0 10 0;
#X connect 9 0 6 0;
#X connect 9 1 7 0;
#X connect 9 2 8 0;
#X connect 10 0 2 0;
#X connect 10 1 3 0;
#X connect 10 2 4 0;
#X connect 10 3 5 0;
