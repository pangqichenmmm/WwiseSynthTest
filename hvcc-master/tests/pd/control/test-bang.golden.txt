[@ 0.000] print: bang
[@ 0.000] print: bang
[@ 0.000] print: bang
[@ 0.000] print: bang
[@ 0.000] print: bang
[@ 0.000] print: bang
[@ 0.000] print: bang
[@ 0.000] print: bang
[@ 0.000] print: bang
[@ 0.000] print: bang
[@ 0.000] print: bang
[@ 0.000] print: bang
